src/ontology/mondo-edit.obo

```diff
@@ -20509,6 +20509,7 @@ property_value: IAO:0000589 "myopia (disease)" xsd:string
 id: MONDO:0001385
 name: cortical blindness
 def: "Visual impairment due to visual cortex dysfunction." [NCIT:P378]
+synonym: "cortical/cerebral visual impairment" EXACT []
```

Comment:
we should have dbrefs for synonyms

Reply:
fixed in https://github.com/monarch-initiative/mondo/pull/7335